### PR TITLE
Update NOTICE.txt

### PR DESCRIPTION
**Pull request by dave2wave**
Properly name projects and products from the Apache Software Foundation. See http://www.apache.org/foundation/marks/guide

### CODE REVIEW

File: NOTICE.txt [hunk 1]
```diff
@@ -23,10 +23,49 @@ This product includes components of the following software:
...
+  Apache Commons Net              Apache License 2.0
+  Apache Commons Pool             Apache License 2.0
+  Apache Commons Pool2            Apache License 2.0
+  Apache Curator                  Apache License 2.0
+  Apache Derby                    Apache License 2.0
   Apache Directory                Apache License 2.0
+  Apache Flume                    Apache License 2.0
+  Apache Geronimo                 Apache License 2.0
+  Apache Hadoop                   Apache License 2.0
+  Apache Hbase                    Apache License 2.0
```
Comment (dimaspivak) [inline NOTICE.txt]: HBase*

File: NOTICE.txt [hunk 2]
```diff
@@ -23,10 +23,49 @@ This product includes components of the following software:
...
+  Apache Hive                     Apache License 2.0
+  Apache HTrace                   Apache License 2.0
+  Apache HttpComponents           Apache License 2.0
+  Apache Ivy                      Apache License 2.0
+  Apache Kafka                    Apache License 2.0
+  Apache Log4J                    Apache License 2.0
```
Comment (dimaspivak) [inline NOTICE.txt]: Log4j

File: NOTICE.txt [hunk 3]
```diff
@@ -23,10 +23,49 @@ This product includes components of the following software:
...
+  Apache Lucene                   Apache License 2.0
+  Apache Parquet                  Apache License 2.0
+  Apache Solr                     Apache License 2.0
+  Apache Spark                    Apache License 2.0
+  Apache Thrift                   Apache License 2.0
+  Apache Velocity                 Apache License 2.0
+  Apache Xerces                   Apache License 2.0
+  Apache Zookeeper                Apache License 2.0
```
Comment (dimaspivak) [inline NOTICE.txt]: ZooKeeper*

File: NOTICE.txt (hunk 1 above)
Comment (dave2wave) [reply NOTICE.txt]: Yes

File: NOTICE.txt (hunk 2 above)
Comment (dave2wave) [reply NOTICE.txt]: Yes

File: NOTICE.txt (hunk 3 above)
Comment (dave2wave) [reply NOTICE.txt]: Yes, thanks

**Comment by dave2wave** [comment]
Requested changes were made

**Comment by dave2wave** [comment]
I'm not clear on what to do next as I've made the requested changes, but cannot seem to resubmit.

**Comment by metadaddy** [comment]
@dave2wave Looks like your changes are automatically reflected here. I think what happens is when you commit to your local, then push to GitHub, it automatically updates the pull request to the tip of your branch. I think.

@dimaspivak I think the case corrections have been made, right?

**Comment by dave2wave** [comment]
The changes were made I remain unclear on what should be done next with this PR?

**Comment by metadaddy** [comment]
@dave2wave I'm merging this into our upstream repository now. Could you send me your email address (I am pat at streamsets dot com) so I can set the commit author appropriately?

**Comment by metadaddy** [comment]
Merged upstream as 7e02c3174cd70361c0bbf4c4972cf2550aa63149

**Pull request closed without merging**